src/FolderConfig.cpp

```diff
@@ -0,0 +1,54 @@
+#include "FolderConfig.h"
+#include <QFile>
+#include <QDebug>
+#include<QDir>
```

Comment:
Need to setup automatic formatter :).

Just a remark. No need to fix.

Reply:
Can we try this one? :)

https://doc.qt.io/qtcreator/creator-beautifier.html

Reply:
It was a note for future.